lib/Cake/Console/Command/Task/ExtractTask.php

```diff
@@ -151,7 +151,7 @@ protected function _getPaths() {
  */
 	public function execute() {
 		if (!empty($this->params['exclude'])) {
-			$this->_exclude = explode(',', $this->params['exclude']);
+			$this->_exclude = explode(',', str_replace('/', DS, $this->params['exclude']));
```

Comment:
I think no. This change is because string comparison by `preg_*`.

Maybe only pass `array_map()` with `realpath()`, so that `array_unique` would filter against full paths.